addons/netfox/properties/property-entry.gd

```diff
@@ -4,6 +4,7 @@ class_name PropertyEntry
 var _path: String
 var node: Node
 var property: String
+var type: Variant.Type ##see typeof()
```

Comment:
This isn't needed for this PR, but its good to have for any future edits. Freely remove these 2 lines.

Reply:
Next time I recommend just not including these in the PR if it's not needed for the feature :slightly_smiling_face: